BankSDK/GiniBankSDK/Documentation/source/Integration.md

```diff
@@ -260,7 +260,6 @@ The components that can be found in the library are:
 * **Review**: Allows to check the quality of one or several images and the ability to delete when multipage is enabled (`ReviewViewController`).
 * **Analysis**: Provides a UI for the analysis process of the document by showing the user capture tips when an image is analyzed or the document information when it is a PDF. In both cases an image preview of the document analyzed will be shown (`AnalysisViewController`).
 * **Help**: Helpful tutorials indicating how to use the open with feature, which are the supported file types and how to capture better photos for a good analysis (`HelpMenuViewController`).
-* **No results**: Shows some suggestions to capture better photos when there are no results after an analysis (`ImageAnalysisNoResultsViewController`).
```

Comment:
[Clarification] Why did we remove it? We still have `No results` screen 😉

Reply:
Here the old code was referring to a VC that is not used anymore: ImageAnalysisNoResultsViewController. The file was also deleted as part of this MR.